### ISSUE TITLE
[Bug] small issue in the vertical navbar link css

### ISSUE THREAD

**Issue by omesh-omg** [layer5io/getnighthawk#350]
#### Current Behavior
in the vertical navbar the 4th doc link [RECENT DISCUSSIONS ON NIGHTHAWK] goes out of the navbar 
![image](https://github.com/layer5io/getnighthawk/assets/64972958/eaf4aed0-67c0-4df3-90de-050d3fc3303b)

#### Expected Behavior
![image](https://github.com/layer5io/getnighthawk/assets/64972958/e1a19b37-f4d6-4e4c-8e1f-a710e24c1527)

#### Environment

- **Host OS:** Mac Linux Windows
- **Platform:** Docker or Kubernetes
- **Meshery Server Version:** stable-v
- **Meshery Client Version:** stable-v

---

#### Contributor Resources

- See the [Contributing to getnighthawk.dev Website](https://github.com/layer5io/getnighthawk/blob/master/CONTRIBUTING.md)
- See the Project Site section of [GetNighthawk project doc](https://docs.google.com/document/d/1lHfMo4iIx2WXFZIspfHyxTsPR1T63_2IV5NUkgxoo0w)
- 🎨 Wireframes and designs for Nighthawk Site in [Figma](https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/Layer5-Designs?type=design&node-id=5046-0&mode=design&t=UYVaGhic2LY8EXee-0)
- Meshery UI Nighthawk (Performance) in [Figma](https://www.figma.com/file/SMP3zxOjZztdOLtgN4dS2W/Meshery-UI?type=design&node-id=2-5814&mode=design&t=Gt14dmRRlpL09apZ-0)
- 🙋🏾🙋🏼 Questions: [Layer5 Discussion Forum](https://discuss.layer5.io) and [Layer5 Community Slack](http://slack.layer5.io)

**Comment by omesh-omg** [comment]
@Savio629 @leecalcote  can u assign this issue to me 👀

**Comment by leecalcote** [comment]
Thank you, gents. 👏

**Comment by abhijeetgauravm** [comment]
![image](https://github.com/layer5io/getnighthawk/assets/65087356/67531c51-570b-42ee-8243-bfe4c0126726)
@omesh-omg I checked in edge and chrome browser, but won't able to reproduce the issue!

**Comment by omesh-omg** [comment]
It happens when reduce the size of window like for responsive @abhijeetgauravm

**Comment by omesh-omg** [comment]
Width of the window
When reduced like for ipad maybe

**Comment by abhijeetgauravm** [comment]
> It happens when reduce the size of window like for responsive @abhijeetgauravm

Yeah I checked at different screen width then i reproduced the issue. Thanks for raising it @omesh-omg !

**Comment by omesh-omg** [comment]
Thanks for mentioning I thought it's useless 😅
